src/App.jsx

```diff
@@ -14,57 +15,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood === '' ? (
+          <>No goods selected</>
```

Comment:
- You don't need to compare with an empty string, just check if a value exists https://developer.mozilla.org/en-US/docs/Glossary/Falsy
- React fragments are redundant here as well

Reply:
if i change the validation it will remove the default value

Reply:
> if i change the validation it will remove the default value

I don't get it, something like this should work
```javascrtipt
{selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
```